### ISSUE TITLE
scheduledJobs name

### ISSUE THREAD

**Issue by ricardofbarros** [node-schedule/node-schedule#54]
For purpose of managing/listing scheduled jobs in an interface, wouldn't it be better if we had an object like this

```javascript
{ '0': 
   { job: [Function],
     name: 'sudo rm -rf /',
     trackInvocation: [Function],
     stopTrackingInvocation: [Function],
     cancel: [Function],
     cancelNext: [Function],
     nextInvocation: [Function],
     _events: {} } }
```

instead of this

```javascript
{ 'sudo rm -rf /': 
   { job: [Function],
     name: 'sudo rm -rf /',
     trackInvocation: [Function],
     stopTrackingInvocation: [Function],
     cancel: [Function],
     cancelNext: [Function],
     nextInvocation: [Function],
     _events: {} } }
```

Basically, having an id on object property, just because in an interface it would be more url-friendly having an id to cancel the job, 

http://example.com/cronjobs/cancel/1

instead of 

http://example.com/cronjobs/cancel/sudo rm -rf /

you feel me ? :D 

If this isn't applicable at least having an option to define the object property, something like this

```javascript
var j = schedule.scheduleJob(['sudo rm -rf /'],rule, function(){
    console.log('The answer to life, the universe, and everything!');
});
```

If the function 'schelduleJob' receive an array it would know we would want a numeric-id alike object property, just tell me if this make any sense, if it doesn't I will have to store the jobs in a database.

**Comment by tejasmanohar** [comment]
since this is a pretty big API change, let's hear what others think! i think it's a quite logical concept. a change like this would definitely be deployed in a major version release.

tagging those who may have thoughts on subject:

@santigimeno 
@mattpat

**Comment by ricardofbarros** [comment]
Hello,
I could help you with a PR refactoring this part of the code, just give me a few days and I will come up with a API usage concept for this case and all the others.

**Comment by tejasmanohar** [comment]
@ricardofbarros I support it. An ID is more useful than the task itself. If you send in a PR altering the tests + API, we'll look over and probably merge it in but for a major release, maybe v0.2.1 probably 0.3 though.

**Comment by tejasmanohar** [comment]
@ricardofbarros Any luck on the PR? If you need any help or come across trouble, link us to the fork and let us know here :+1:

**Comment by DaAwesomeP** [comment]
Ideally it should be able to grab the schedule object based on ID and then be able to modify it in any way (change values, cancel, etc.).

Make sure to leave an option when creating tasks to automatically find the next ID.  However, if tasks are deleted/cancelled, it should not use the same ID as the now non-existent one. A simple counter variable should work.

**Comment by jonhester** [comment]
Fixed in #130. Thanks @ricardofbarros!
